form_app/lib/src/validation.dart

```diff
@@ -114,7 +114,7 @@ class _FormValidationDemoState extends State<FormValidationDemo> {
                 // checkbox.
                 FormField(
                   initialValue: false,
-                  validator: (value) {
+                  validator: (dynamic value) {
```

Comment:
Are you getting an error without this `dynamic`? I feel like it shouldn't be necessary, but I'm willing to be corrected. 😄